### ISSUE TITLE
About loading model mods into the gameAbout loading model mods into the game

### ISSUE THREAD

**Issue by yequ172672** [Kaplas80/GuJian3Manager#29]
Hello, how should I correctly load the modified model file into the game, I try to import the modified model into the game. Its coordinates, orientation, and texture are not displayed correctly, I merged an extra box to the hair mesh

**Comment by Kaplas80** [comment]
I'm sorry, but I haven't tried to replace 3D models. I made this app just for translating texts and the file decryption and model extracting was just a "side effect" 😅

There is a video in #24 that I think it shows how to replace models, but it is in Chinese and I don't understand it. Maybe @stbz4088 can help you.

**Comment by yequ172672** [comment]
ok，thanks

**Comment by stbz4088** [comment]
I think this tool can theoretically replace anything and I have successfully used it to replace the character's outfit. If you know Chinese, maybe my video will help. https://www.bilibili.com/video/BV1bF41137Xw

**Comment by yequ172672** [comment]
> 

能换但骨骼材质骨骼缩放都是大问题，老引擎也找不到开发文档

**Comment by yequ172672** [comment]
点错给关了，我试过增加模型，但进去是白模方向也不对

**Comment by stbz4088** [comment]
> > 
> 
> 能换但骨骼材质骨骼缩放都是大问题，老引擎也找不到开发文档

骨骼替换就不清楚了,我不懂游戏开发,我只想追加几套外装,但最终只实现了替换外装的效果

**Comment by yequ172672** [comment]
想换衣服，加首饰之类的需要搞明白骨骼怎么整，单纯换贴图的画工具已经很完善了

**Comment by yequ172672** [comment]
The third-party model has been successfully loaded into the game, and the animation is normal

**Comment by yequ172672** [comment]
[https://www.bilibili.com/video/BV18N411N7nD/?spm_id_from=333.999.0.0&vd_source=47d27fa2ecdd1787832ab256b30864f9](url)

**Comment by Hexaae** [comment]
> I made this app just for translating texts and the file decryption and model extracting was just a "side effect" 😅

I'm interested in doing an Italian translation... So it would be possible to translate the WHOLE game with this tool?

**Comment by yequ172672** [comment]
> > I made this app just for translating texts and the file decryption and model extracting was just a "side effect" 😅
> 
> I'm interested in doing an Italian translation... So it would be possible to translate the WHOLE game with this tool?

Yes it can be done, you probably need this: https://github.com/eprilx/Gujian3TextEditor

**Comment by Kaplas80** [comment]
> > I made this app just for translating texts and the file decryption and model extracting was just a "side effect" 😅
> 
> I'm interested in doing an Italian translation... So it would be possible to translate the WHOLE game with this tool?

Yes, you should be able to translate it with this tool, but you'll also need alanm's mod loader that was stored in zenhax.com and it is now offline... Maybe someone in this thread has it and can upload somewhere.

**Comment by Kaplas80** [comment]
Ok, I've found it.

This is the source code: https://github.com/alanm20/Gujian3TextMod
This is the download link: https://www.mediafire.com/file/cpp100wbd3v5p2o/G3TextMod_0_2.zip/file
